test/jdk/javax/swing/JColorChooser/8065098/JColorChooserDnDTest.java

```diff
@@ -34,21 +34,17 @@
 public class JColorChooserDnDTest {
```

Comment:
Missing @test and @run tags?

Reply:
This part of the test doesn't run on jtreg so I don't think this is applicable.